dnn/variational_with_real_data.ipynb

```diff
@@ -7,23 +7,23 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.        var_output = keras.layers.Dense(1, name='var_output',
</pre><p>This seems to be missing the numerical stability fix from the blog post we reviewed during the group pair programming</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Ack, updated.